components/PostContent/PostContent.tsx

```diff
@@ -15,53 +15,54 @@ export default function PostContent( props: {
 			{
 				props.blocks.map( ( block, i ) => {
 					const blockProps = mapAttributesToProps( block.attributes || [] );
-					const key = `block-${i}`;
+					const defaultProps = { key: `block-${i}` };
 
 					switch ( block.name ) {
 						case 'core/image':
+							const imageProps = {
```

Comment:
This isn't really addressing the fact that `srcSet` was removed. All that this is doing is outsmarting the type checker. I'm really not sure how, though. 🤔